lemmy_apub/src/activity_queue.rs

```diff
@@ -28,6 +28,11 @@ use serde::{export::fmt::Debug, Deserialize, Serialize};
 use std::{collections::BTreeMap, future::Future, pin::Pin};
 use url::Url;
 
+/// Sends a local activity to a single, remote actor.
+///
+/// * `activity` the apub activity to be sent
+/// * `creator` the local actor which created the activity
+/// * `inbox` the inbox url where the activity should be delivered to
 pub async fn send_activity_single_dest<T, Kind>(
   activity: T,
   creator: &dyn ActorType,
```

Comment:
I was considering adding an assert in these methods to ensure that the sender is always a local actor.

Reply:
Remember no asserts, just either boolean checks or results, because assert means it will panic and potentially crash the server.

Reply:
`debug_assert` would be fine, but then we need to make test builds in debug mode. Plus I think our federation tests will just restart Lemmy if it crashes.